packages/ibm-products-web-components/src/components/interstitial-screen/interstitial-screen.stories.ts

```diff
@@ -0,0 +1,503 @@
+    let currentStep = 0;
+
+    const handleAction = async (actionType) => {
+      const carouselAPI = config?.carouselAPI;
+
+      if (!carouselAPI) {
+        return;
+      }
+
+      if (actionType === 'next') {
+        carouselAPI.next();
+        currentStep = currentStep + 1;
+      } else if (actionType === 'back') {
+        carouselAPI.prev();
+        currentStep = currentStep - 1;
+      } else {
+        // handle 'skip' or 'submit' action here
+        document
+          .querySelector('c4p-interstitial-screen')
+          ?.toggleAttribute('open');
+      }
+
+      rerender();
+    };
+
+    const handleSkip = () => handleAction('skip');
+    const handleStart = () => handleAction('start');
+    const handleClickNext = () => handleAction('next');
+    const handleClickPrev = () => handleAction('back');
+
+    const rerender = () => {
+      litRender(
+        html`
+          <div id="page-content-selector">
+            <cds-button @click="${toggleButton}"
+              >Show Interstitial modal</cds-button
+            >
+          </div>
+          <c4p-interstitial-screen
+            ?fullscreen=${false}
+            ?open=${true}
+            @c4p-interstitial-opened=${(e) => {
+              if (!initialized) {
+                config = e.detail;
+                initialized = true;
+                rerender();
+              }
+            }}
+          >
+            <c4p-interstitial-screen-header
+              header-title="Use case-specific title"
+              header-subtitle="Use case-specific sub title"
+            ></c4p-interstitial-screen-header>
+
+            <c4p-interstitial-screen-body>
+              ${getMultipleContent()}
+            </c4p-interstitial-screen-body>
+
+            <c4p-interstitial-screen-footer>
+              <style>
+                ${footerStyles} cds-button {
+                  block-size: 100%;
+                  display: contents;
+                }
+              </style>
+
+              <div class="${blockClass}--footer">
+                <cds-button
+                  class="${blockClass}--skip-btn"
+                  kind="ghost"
+                  size="lg"
+                  @click=${handleSkip}
+                >
+                  Skip
+                </cds-button>
+
+                <div class="${blockClass}--footer-controls">
+                  ${stepLength > 0 && currentStep > 0
+                    ? html`
+                        <cds-button
+                          class="${blockClass}--prev-btn"
+                          kind="secondary"
+                          size="lg"
+                          title="Back"
+                          @click=${handleClickPrev}
+                        >
+                          Back
+                        </cds-button>
+                      `
+                    : nothing}
+                  ${stepLength > 0 && currentStep < progStepCeil
+                    ? html`
+                        <cds-button
```

Comment:
<img width="1038" height="417" alt="image" src="https://github.com/user-attachments/assets/3c7ba0df-ba05-428b-a45f-b0ee07c9e06e" />

The next button here is a bit shorter than typical. Can we add some story styles to make the sizing feel consistent?